### PR TITLE
[Querly] Support `config` option

### LINKED ISSUE #823: [Querly] Add `config` option
```shell-session
$ querly help check
Usage:
  querly check [paths]

Options:
  [--config=CONFIG]
                     # Default: querly.yml
  [--root=ROOT]
  [--format=FORMAT]
                     # Default: text
                     # Possible values: text, json
  [--rule=RULE]
  [--threads=N]
                     # Default: 8

Check paths based on configuration
```

See also:

https://github.com/sider/runners/blob/f1b7a0277346a23b8eb41a84577ba7a92ad35647/lib/runners/processor/querly.rb#L6

### PR DESCRIPTION
**Pull request by ybiquitous**
> Explain a summary, purpose, or background for this change.

This change adds the `config` option for `sider.yml`. Users can specify any config file they want via the option.

E.g.

```yaml
linter:
  querly:
    config: .config/querly.yml
```

> Link related issues, e.g. "Fix #<ISSUE_ID>", "Related to #<ISSUE_ID>", or "None."

Fix #823

> Check the following items.

- [x] Add a new [changelog](https://github.com/sider/runners/blob/master/CHANGELOG.md) entry if this change is notable.

### CODE REVIEW

File: lib/runners/processor/querly.rb [hunk 1]
```diff
@@ -29,69 +33,87 @@ def analyzer_version
     end
 
     def setup
-      ret = install_gems default_gem_specs, optionals: OPTIONAL_GEMS, constraints: CONSTRAINTS do
+      install_gems default_gem_specs, optionals: OPTIONAL_GEMS, constraints: CONSTRAINTS do
         yield
       end
-
-      # NOTE: Exceptionally MissingFileFailure is treated as successful
-      if ret.instance_of? Results::MissingFilesFailure
```
Comment (ybiquitous) [inline lib/runners/processor/querly.rb:37]: [note] I will remove `Results::MissingFilesFailure` via #1121.

File: lib/runners/processor/querly.rb [hunk 2]
```diff
@@ -29,69 +33,87 @@ def analyzer_version
...
-        next unless match
-        match[1]
-      end.compact
+    def config_file
+      config_linter[:config]
+    end
+
+    def default_config_file
+      return @default_config_file if defined? @default_config_file
+      @default_config_file ||= Dir.glob("querly.{yml,yaml}").first
```
Comment (yykamei) [inline lib/runners/processor/querly.rb:93]: How about adding a warning in case users have the two configuration files?

Comment (ybiquitous) [reply lib/runners/processor/querly.rb:93]: Looks good. I'll add it.

Comment (ybiquitous) [reply lib/runners/processor/querly.rb]: Fixed via d19bdd41

**Comment by ybiquitous** [comment]
Thank you! 🥰

**Pull request merged**